src/main/java/frc/robot/subsystems/turretSubsystem/TurretSubsystem.java

```diff
@@ -441,6 +441,14 @@ public void aimTurret() {
           tagDistanceEntry.log(distanceToTag);
         }
 
+        if (visionSubsystem.getTurretPitch(tagID) == 0) {
+          setPitchPos(Constants.TurretConstants.defaultShotPitch);
+          setTurretPos(0);
```

Comment:
Even though I imagine that we won't have a different default rotation than straight ahead, let's make this a constant as well.